lib/util/util.inherit.js

```diff
@@ -52,10 +50,13 @@ this.util = this.util || {};
 			throw new Error( 'Bad function name given. At least one word character or $ required.' );
 		}
 
-		eval( 'namedFn = function ' + fnName +
-			'(){ evilsSeed.apply( this, arguments ); }' );
+		// No, none of these functions is superfluous! Before this used to fail with a
+		// "ReferenceError: evilsSeed is not defined" when Shift+reloading in Firefox.
+		// Source: http://stackoverflow.com/a/22880379
+		namedFn = ( new Function( 'return function( fn ) { return function ' + fnName +
+			'() { return fn( this, arguments ); }; };' )() )( Function.apply.bind( evilsSeed ) );
```

Comment:
It was when I played around with my version. Since this is hard/impossible to test I'm afraid to change my version (the only one that works for sure, as far as I know) to something else.